src/pytorch_kinematics/transforms/rotation_conversions.py

```diff
@@ -485,6 +538,8 @@ def axis_angle_to_matrix(axis_angle):
     Returns:
         Rotation matrices as tensor of shape (..., 3, 3).
     """
+    warn('This is deprecated because it is slow. Use axis_and_angle_to_matrix',
```

Comment:
Isn't this called `axis_and_angle_to_matrix_33` and `axis_and_angle_to_matrix_44` instead of just `axis_and_angle_to_matrix` now?